composer.json

```diff
@@ -20,13 +24,10 @@
         "phpmd/phpmd": "^2.6",
         "phan/phan": "0.12.x"
     },
-    "scripts": {
-      "pre-install-cmd": "mkdir -p project/libraries"
-    },
     "autoload" : {
         "psr-4": {
             "LORIS\\": "src/"
         },
-        "classmap": ["project/libraries", "php/libraries", "php/exceptions"]
+        "classmap": ["php/libraries", "php/exceptions"]
```

Comment:
The "project" hack had to be removed here or `composer update` dies with the error:

```
                                                                               
  [RuntimeException]                                                           
  Could not scan for classes inside "/Users/driusan/Code/Loris-InstrumentMana  
  ger/vendor/aces/loris/project/libraries" which does not appear to be a file  
   nor a folder                                                                
                                                                               

update [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-dev] [--lock] [--no-custom-installers] [--no-autoloader] [--no-scripts] [--no-progress] [--no-suggest] [--with-dependencies] [--with-all-dependencies] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--ignore-platform-reqs] [--prefer-stable] [--prefer-lowest] [-i|--interactive] [--root-reqs] [--] [<packages>]...
```